formapp/app.py

```diff
@@ -32,7 +32,7 @@
 # 3 - anyone
 # 4 - banned
 
-NEWACCOUNTPERMISSIONS = 3
+NEWACCOUNTPERMISSIONS = 2
```

Comment:
Please change this back to 3. By default, if the permissions are set to student anyone can make any number of accounts and spam polls or dodge bans.